### ISSUE TITLE
Bug in read file Amazon

### ISSUE THREAD

**Issue by MojtabaSaif** [SSRSGJYD/LPANNI#1]
Hi
I try to run the program but I give a runtime error!
I use the default data set Amazon.
Please see line 8 of the Amazon. This graph has a node with the name 444232. 
The max name for the node of Amazon should be 334863 in your program.
You must create a difference between the index of the node with the name of a node in the Data set file.

**Comment by SSRSGJYD** [comment]
You are right, there should be a mapping between the index and the name of a node. To make it simple, I just set the node size to a bigger number, i.e, 550000 at https://github.com/SSRSGJYD/LPANNI/blob/811dbb14f7e17b96d4fb16982c2142e3265d8aad/src/LPANNI/graph.cpp#L57.

**Comment by MojtabaSaif** [comment]
Hi
I suggest you rewrite the program. The mapping between the index and the name of a node is good.
